--- ./mail/build.mk	2012-05-09 23:28:42.000000000 +0200
+++ ./mail/build.mk-new	2012-05-09 23:38:58.000000000 +0200
@@ -51,6 +51,8 @@ tier_app_dirs += $(APP_LIBXUL_DIRS)
 endif
 
 TIERS += app
+ 
+tier_app_dirs += xpfe/components/autocomplete
 
 ifdef MOZ_EXTENSIONS
 tier_app_dirs += extensions

el Makefile
 
 TIERS += app
 

--- ./suite/build.mk	2012-05-10 00:16:50.000000000 +0200
+++ ./suite/build.mk-new	2012-05-10 00:22:18.000000000 +0200
@@ -45,6 +45,8 @@ include $(topsrcdir)/toolkit/toolkit-tie
 endif
 
 TIERS += app
+ 
+tier_app_dirs += xpfe/components/autocomplete
 
 ifdef MOZ_EXTENSIONS
 tier_app_dirs += extensions
