--- comm-1.9.1/mailnews/base/prefs/content/AccountManager.js.bak	2010-10-27 20:39:14.000000000 +0200
+++ comm-1.9.1/mailnews/base/prefs/content/AccountManager.js	2010-11-14 18:55:31.297159154 +0100
@@ -1190,7 +1190,7 @@
         var entryName = catEnum.getNext().QueryInterface(string).data;
         var svc = Components.classes[catMan.getCategoryEntry(CATEGORY, entryName)]
                             .getService(Ci.nsIMsgAccountManagerExtension);
-        if (svc.showPanel(server))
+        if (svc.showPanel(server) && (svc.name != "enigprefs"))
 {
           var sbs = Components.classes["@mozilla.org/intl/stringbundle;1"]
                               .getService(Ci.nsIStringBundleService);
