--- mozilla/editor/ui/composer/content/editor.xul.nodebug	2003-10-24 14:13:43.000000000 +0200
+++ mozilla/editor/ui/composer/content/editor.xul	2004-05-12 14:30:51.381321567 +0200
@@ -164,7 +164,7 @@
     <menu id="windowMenu"/>
 
     <!-- DEBUG only -->
-    <menu id="debugMenu" /> 
+    <!--<menu id="debugMenu" /> -->
     <!-- end DEBUG only -->
   
     <!-- help menu filled from globalOverlay -->
--- mozilla/editor/ui/composer/content/editorOverlay.xul.nodebug	2004-03-25 13:42:09.000000000 +0100
+++ mozilla/editor/ui/composer/content/editorOverlay.xul	2004-05-12 14:30:51.382321578 +0200
@@ -899,7 +899,7 @@
   <menuitem id="menu_pasteQuote" accesskey="&editpastequotation.accesskey;" command="cmd_pasteQuote" key="pastequotationkb"/>
 
   <!-- DEBUG only -->
-  <menu id="debugMenu" label="&debugMenu.label;">
+ <!-- <menu id="debugMenu" label="&debugMenu.label;">
   <menupopup>
     <menuitem label="&newEditorTestPage.label;" oncommand="window.openDialog('chrome://editor/content','_blank','chrome,all,dialog=no','chrome://editor/content/EditorInitPage.html')"/>
     <menuitem label="&textEditorCmd.label;"     oncommand="EditorNewPlaintext();" />
@@ -969,6 +969,7 @@
             oncommand="window.focus()"/>
   </menupopup>
   </menu>
+  -->
   <!-- end DEBUG only -->
 
 </overlay>
--- mozilla/editor/ui/composer/content/TextEditorAppShell.xul.nodebug	2003-10-24 14:13:43.000000000 +0200
+++ mozilla/editor/ui/composer/content/TextEditorAppShell.xul	2004-05-12 14:30:51.383321590 +0200
@@ -113,7 +113,7 @@
       <menu accesskey="t" id="tasksMenu"/>
 
       <!-- DEBUG only -->
-      <menu id="debugMenu" />
+      <!--<menu id="debugMenu" />-->
       <!-- end DEBUG only -->
 
       <!-- help menu filled from globalOverlay -->
--- mozilla/xpfe/browser/resources/content/navigatorOverlay.xul.nodebug	2004-02-29 00:36:40.000000000 +0100
+++ mozilla/xpfe/browser/resources/content/navigatorOverlay.xul	2004-05-12 14:30:51.384321601 +0200
@@ -457,6 +457,7 @@
   <menu accesskey="&helpMenuCmd.accesskey;" id="menu_Help"/>
 
   <!-- Menu for testing. -->
+<!--
   <menu id="debugMenu" accesskey="U" label="Debug">
     <menupopup>
       <menu label="Verification">
@@ -585,6 +586,7 @@
       <menuitem label="Latest Builds" oncommand="loadURI('http://ftp.mozilla.org/pub/mozilla.org/mozilla/nightly/latest/');"/>
     </menupopup>
   </menu>
+-->
 </menubar>
 
 </overlay>
--- mozilla/xpfe/components/prefwindow/resources/content/preftree.xul.nodebug	2003-12-31 08:13:12.000000000 +0100
+++ mozilla/xpfe/components/prefwindow/resources/content/preftree.xul	2004-05-12 14:33:13.933953282 +0200
@@ -164,7 +164,7 @@
         </treeitem> 
       </treechildren>
     </treeitem>
-
+<!--
     <treeitem container="true" id="debugItem">
       <treerow>       
         <treecell url="chrome://communicator/content/pref/pref-debug.xul" label="&debug.label;"/> 
@@ -182,6 +182,7 @@
         </treeitem>
       </treechildren>
     </treeitem>
+    -->
 
     <!-- commented out until bug 82487 is fixed -->
     <!--
