--- mozilla/modules/libpref/src/init/all.js.font	2004-08-11 11:36:54.123867157 +0200
+++ mozilla/modules/libpref/src/init/all.js	2004-08-11 11:43:33.339977705 +0200
@@ -1565,17 +1565,21 @@
 pref("font.name.sans-serif.x-cyrillic", "cronyx-helvetica-koi8-r");
 pref("font.name.monospace.x-cyrillic", "cronyx-courier-koi8-r");
 
-pref("font.name.serif.x-unicode", "adobe-times-iso8859-1");
-pref("font.name.sans-serif.x-unicode", "adobe-helvetica-iso8859-1");
-pref("font.name.monospace.x-unicode", "adobe-courier-iso8859-1");
+pref("font.name.serif.x-unicode", "Times New Roman");
+pref("font.name.sans-serif.x-unicode", "Arial");
+pref("font.name.monospace.x-unicode", "Andale Mono");
+pref("font.name.cursive.x-unicode", "Times New Roman");
+pref("font.name.fantasy.x-unicode", "URW Chancery L");
 
 pref("font.name.serif.x-user-def", "adobe-times-iso8859-1");
 pref("font.name.sans-serif.x-user-def", "adobe-helvetica-iso8859-1");
 pref("font.name.monospace.x-user-def", "adobe-courier-iso8859-1");
 
-pref("font.name.serif.x-western", "adobe-times-iso8859-1");
-pref("font.name.sans-serif.x-western", "adobe-helvetica-iso8859-1");
-pref("font.name.monospace.x-western", "adobe-courier-iso8859-1");
+pref("font.name.serif.x-western", "Times New Roman");
+pref("font.name.sans-serif.x-western", "Arial");
+pref("font.name.monospace.x-western", "Andale Mono");
+pref("font.name.cursive.x-western", "Times New Roman");
+pref("font.name.fantasy.x-western", "URW Chancery L");
 
 pref("font.name.serif.zh-CN", "isas-song ti-gb2312.1980-0");
 pref("font.name.sans-serif.zh-CN", "isas-song ti-gb2312.1980-0");
