--- comm-1.9.1/mozilla/modules/libpref/src/init/all.js.lang	2009-11-17 17:39:46.819409105 +0100
+++ comm-1.9.1/mozilla/modules/libpref/src/init/all.js	2009-11-17 17:53:45.322458470 +0100
@@ -890,7 +890,7 @@
 pref("intl.charset.detector",               "chrome://global/locale/intl.properties");
 pref("intl.charset.default",                "chrome://global-platform/locale/intl.properties");
 pref("intl.ellipsis",                       "chrome://global-platform/locale/intl.properties");
-pref("intl.locale.matchOS",                 false);
+pref("intl.locale.matchOS",                 true);
 // fallback charset list for Unicode conversion (converting from Unicode)
 // currently used for mail send only to handle symbol characters (e.g Euro, trademark, smartquotes)
 // for ISO-8859-1
