--- mozilla/modules/libpref/src/init/all.js.print	2004-08-11 11:47:48.120256848 +0200
+++ mozilla/modules/libpref/src/init/all.js	2004-08-11 11:48:04.873932710 +0200
@@ -1773,7 +1773,7 @@
 
 /* PostScript print module prefs */
 // pref("print.postscript.enabled",      true);
-pref("print.postscript.paper_size",    "letter");
+pref("print.postscript.paper_size",    "a4");
 pref("print.postscript.orientation",   "portrait");
 pref("print.postscript.print_command", "lpr ${MOZ_PRINTER_NAME:+'-P'}${MOZ_PRINTER_NAME}");
 
