--- comm-1.9.1/mozilla/modules/libpref/src/init/all.js.xvt	2009-10-25 15:33:53.000000000 +0100
+++ comm-1.9.1/mozilla/modules/libpref/src/init/all.js	2009-11-17 17:39:46.819409105 +0100
@@ -2336,10 +2336,10 @@
 pref("java.default_java_location_solaris", "/usr/j2se");
 pref("java.default_java_location_others", "/usr/java");
 pref("java.java_plugin_library_name", "javaplugin_oji");
-pref("applications.telnet", "xterm -e telnet %h %p");
-pref("applications.tn3270", "xterm -e tn3270 %h");
-pref("applications.rlogin", "xterm -e rlogin %h");
-pref("applications.rlogin_with_user", "xterm -e rlogin %h -l %u");
+pref("applications.telnet", "xvt -e telnet %h %p");
+pref("applications.tn3270", "xvt -e tn3270 %h");
+pref("applications.rlogin", "xvt -e rlogin %h");
+pref("applications.rlogin_with_user", "xvt -e rlogin %h -l %u");
 pref("print.print_command", "lpr ${MOZ_PRINTER_NAME:+-P\"$MOZ_PRINTER_NAME\"}");
 pref("print.printer_list", ""); // list of printers, separated by spaces
 pref("print.print_reversed", false);
