--- firefox-47.0/mozilla/toolkit/system/unixproxy/nsLibProxySettings.cpp.0042~	2016-06-07 15:05:15.236231597 +0200
+++ firefox-47.0/mozilla/toolkit/system/unixproxy/nsLibProxySettings.cpp	2016-06-07 15:05:52.723025518 +0200
@@ -7,6 +7,7 @@
 #include "mozilla/ModuleUtils.h"
 #include "nsIServiceManager.h"
 #include "nsIURI.h"
+#include "nsNetCID.h"
 #include "nsString.h"
 #include "nsCOMPtr.h"
 #include "nsNetCID.h"
