--- a/mozilla/configure.orig	2014-06-11 15:49:48.304505882 +0100
+++ a/mozilla/configure	2014-06-11 15:50:32.480503301 +0100
@@ -21737,7 +21737,7 @@
   fi
 
 
-if test -n "${JAVA_BIN_PATH}" -o \
+if test -n "${JAVA_BIN_PATH}" -a \
   \( "$OS_TARGET" = Android -a x"$MOZ_WIDGET_TOOLKIT" != x"gonk" \); then
   if test -z "$JAVA" -o "$JAVA" = ":"; then
     { echo "configure: error: The program java was not found.  Set \$JAVA_HOME to your Java SDK directory or use --with-java-bin-path={java-bin-dir}" 1>&2; echo "configure: error: The program java was not found.  Set \$JAVA_HOME to your Java SDK directory or use --with-java-bin-path={java-bin-dir}" 1>&5; exit 1; }
