--- seamonkey-1.0.3/mozilla/mailnews/mime/src/mimemult.cpp.bak	2005-09-14 18:11:13.000000000 +0200
+++ seamonkey-1.0.3/mozilla/mailnews/mime/src/mimemult.cpp	2006-07-30 10:20:27.531128548 +0200
@@ -73,7 +73,6 @@
 extern "C" MimeObjectClass mimeMultipartRelatedClass;
 extern "C" MimeObjectClass mimeMultipartSignedClass;
 extern "C" MimeObjectClass mimeInlineTextVCardClass;
-extern "C" MimeExternalObjectClass mimeExternalObjectClass;
 
 #if defined(DEBUG) && defined(XP_UNIX)
 static int MimeMultipart_debug_print (MimeObject *, PRFileDesc *, PRInt32);
